infra/base-images/base-clang/checkout_build_install_llvm.sh

```diff
@@ -151,8 +140,8 @@ cmake_llvm $CMAKE_EXTRA_ARGS \
     -DCMAKE_INSTALL_PREFIX=/usr/msan/ \
     -DCMAKE_CXX_FLAGS="-fsanitize-blacklist=$WORK/msan/blacklist.txt"
 
-ninja -j $NPROC cxx
-ninja install-cxx
+ninja cxx
```

Comment:
Why is -j moved from build step on this line to install-cxx step ? install shouldnt need -j right ?